### ISSUE TITLE
Some site tweaks to increase social engagement

### ISSUE THREAD

**Issue by robby-dermody** [monero-project/monero-site#1459]
I'm proposing a few site tweaks to increase social engagement. I think it's important that social media engagement options be more prominent on the site because social engagement (i.e. users following the Monero twitter, Facebook, joining a Monero chat) is the engagement method that the vast, vast majority (90%+?) of users will do over anything else (and normally to the exclusion of anything else). We should make it easy for them, as this majority of users is what powers social media presence and social media outreach.

Here's what I propose:

1. **Home page** - Add "Join our hangouts" or similar text as the first subsection under the "Join the Community" header, with appropriate text and a link to the https://www.getmonero.org/community/hangouts/ page.

2. **Home page** - Also consider putting a line of icons for social channels on the bottom of the footer of the Monero main page.  Having the logos on the bottom of the page is an established website pattern that allows people to quickly find them without having to even read the page (yes, I am saying that :p).

3. **On "Hangouts" page** - I am aware that we are getting a Monero Discord set up. Once this is done, I propose we change "IRC Channels" to "Chat Channels" (as well as make the relevant body text changes). Consider linking to Discord for the channel names instead of IRC (and have IRC as one of the logos to the right of the channel name). Or, minimally, add the Discord icon to that section, as well as to the list of icons at the top. My reasoning is that Discord seems to currently be the (or one of the) most well-established chat options in the crypto community. As all of these venues are relayed, we should feature the most well-known venue (that the majority of users already have the software for) first.

4. **On "Hangouts" page** - Put the Chat Channels section before the Workgroup Resources and Mailing Lists sections.

5. **On "Hangouts" page** - Add links to Telegram chat, Twitch and Discord chat on the top line of icons, once available. Telegram could be totally disconnected from the IRC channels to avoid spam/idiots (but should be moderated). Twitch and Discord can probably be bridged?

**Comment by erciccione** [comment]
> 1.

That's a good idea.

> 2.

As said by chat: putting social icons in the footer sort of implies those are official social medias accounts of the Monero project. There is no such thing as an official account of the project. So, i disagree with the change.

> 3.

Discord is closed source and not privacy-friendly. If people want to list discord or other chats, i'm ok with that, but i would do it in a separate box, where we can warn users about the privacy implications.

> 4.

I don't see the point in changing this. The first two sections are very small. You can always see that there is something else below.

> 5.

I disagree about adding the Telegram chat. it was chosen to only display it in the footer, because of the high level of spam and low quality of the conversations. I don't think that should be changed. Same for Discord (see my answer to point 3). I don't see the reason for adding twitch and i have no diea who is behind that in the first place.

**Comment by SamsungGalaxyPlayer** [comment]
Regarding 3 - I highly doubt that someone is going to suddenly start using Discord because of the presence of the link on the page. Most likely, an existing Discord user will go "hey look, that's easy to join!" and click the button.

People who care about privacy should be encouraged to use the Matrix server. So while I think we should promote the Matrix server the most, I also don't want to infringe on some existing Discord user joining.

**Comment by robby-dermody** [comment]
Closing this, as we will be opening up a separate ticket with a specific proposed design.
